mypy/dmypy_server.py

```diff
@@ -39,7 +39,8 @@ def daemonize(func: Callable[[], None], log_file: Optional[str] = None) -> int:
     subprocess killed by signal.
     """
     # See https://stackoverflow.com/questions/473620/how-do-you-create-a-daemon-in-python
-    if sys.platform == 'win32':
+    # mypyc doesn't like unreachable code, so trick mypy into thinking the branch is reachable
+    if sys.platform == 'win32' or 0 == 1:
```

Comment:
Ultra minor nit: I would write `bool()` here and below. (Although even better I would switch to checking unreachable code, but this is a separate question.)